### DISCUSSION TITLE
resources/

### DISCUSSION THREAD

**Discussion by giscus** [zxuqian/zxuqian.cn#75] [Announcements]
# resources/

本页收集了日常开发中经常会使用到的前端开发、UI 设计、软件工具等资源（持续整理中）。

https://zxuqian.cn/resources/

**Comment by lkzwc** [comment]
hhh

**Comment by fantingsheng** [comment]
博主意愿加下友链吗？https://spacexcode.com/

**Comment by TimSpan** [comment]
这个 Docusaurus  改造的牛啊

**Comment by zc16607** [comment]
请问一下你之前写的一个js文件，可以测量网页上元素的大小间距的工具，地址在哪里呢。感觉挺好用的，之前是放在书签栏的，后面重装系统之后找不到了。

**Comment by TimSpan** [comment]
啊？

---- Replied Message ----
| From | ***@***.***> |
| Date | 11/05/2024 15:57 |
| To | ***@***.***> |
| Cc | TimSpan ***@***.***>,
Comment ***@***.***> |
| Subject | Re: [zxuqian/zxuqian.cn] resources/ (Discussion #75) |

请问一下你之前写的一个js文件，可以测量网页上元素的大小间距的工具，地址在哪里呢。感觉挺好用的，之前是放在书签栏的，后面重装系统之后找不到了。

**Reply by zc16607** [reply]
嗯？
